diff -Nur old/changes new/changes
--- old/changes	ignored date
+++ new/changes	ignored date
@@ -1,5 +1,6 @@
 unreleased:
  * Some change
+ * Adding feature A
 
 version 0.1:
  * Initial Version
diff -Nur old/src/code new/src/code
--- old/src/code	ignored date
+++ new/src/code	ignored date
@@ -1,3 +1,3 @@
 This is the code.
-It has some lines.
+It has some lines that we change to implement feature A.
 That we want to edit.
